test-dataset-plan.md

```diff
@@ -15,9 +15,9 @@ As mentioned above, we want the test dataset to contain an equal amount of
 data from each of the following tasks:
 
 - Danish web data (OSCAR-da)
-- Danish social media data (HOPE Twitter and Reddit-da)
+- Danish social media data (Twitter, General Discussions from DAGW and Reddit-da)
```

Comment:
You said you wanted to split up Twitter and Reddit-da, but they're still together?